### ISSUE TITLE
UI Component - Tab

### ISSUE THREAD

**Issue by cammellos** [status-im/status-react#13545]
This issue is to track the creation of the component: **Tab**

You can access the link at https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=57%3A13215.

Please have a chat with the design team before starting with the component, and don't forget to add sizing to the issue

**Comment by ibrkhalil** [comment]
Turns out this component is already done
